docs/examples/hypervisors/kvm/README.md

```diff
@@ -0,0 +1,109 @@
+KVM Virtualization
+==================
+
+This sample uses Terraform to demonstrate the Bring Your Own Hypervisor model to deploy custom KVM based Virtual Machines on top of a KVM hypervisor on Oracle Cloud Infrastructure.
+
+In this example we demonstrate the automatic installation of a [Web Application Firewall - WAF](https://en.wikipedia.org/wiki/Web_application_firewall) running on Oracle Cloud Infrastructure (OCI) based on the network topology described on the diagram below, but it can be easily adapted to automate the installation of any Virtual Machine based on KVM:
```

Comment:
Entirely possible that I'm missing something, but doesn't the WAF installation depend on the image used?

Reply:
We only need to attach a single vNIC to the guest, multiple vNICs will be specific to use case and niche, we should keep this as simple as possible. The process to add a second vNIC is documented somewhere, include a link. 

Get an OL4 image PAR from a bucket that will persist and include it in the example along with anything else required so the customer can apply the TF config then connect to the guest _x_ minutes later.